src/index.html

```diff
@@ -15,8 +15,79 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+
+    />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav>
+        <ul>
+          <img
+            class="img"
+            src="/src/images/logo.png"
+            alt="img"
+          />
+          <a
+            class="list"
+            href="Apple"
+          >
+            Apple
+          </a>
```

Comment:
The `a` tag should not be a direct child of the `ul` tag. Please, wrap each `a` tag in an `li` tag. This applies to all `a` tags inside the `ul` tag.